web/auth/user_from_auth.ex

```diff
@@ -109,7 +122,7 @@ defmodule PhoenixGuardian.UserFromAuth do
         if authorization.token == auth.credentials.token do
           authorization
         else
-          {:error, :token_mismatch}
+          {:error, :"Token mismatch"}
```

Comment:
This will be easier to match on if it remains `:token_mismatch`. Putting spaces in atoms is not cool.